dev/ctia/dev/split_tests.clj

```diff
@@ -204,7 +204,7 @@
                                              "\nOut:\n" out
                                              "\nErr:\n" err)
                                         {:res res}))))
-          timeout-seconds 120
+          timeout-seconds 240
```

Comment:
I saw you had some issues with the CI, do we still need to increase this timeout?

Reply:
Yes, I believe for the time being this is unfortunately the best we could do. Unless @frenchy64 thinks differently, I'd like to increase the timeout.

Reply:
Yes, we must increase the limit. master builds are failing.